app/src/main/java/com/example/vmedvediev/redditapp/comments/CommentsActivity.kt

```diff
@@ -54,10 +55,13 @@ class CommentsActivity : AppCompatActivity() {
 
          setupImageLoader()
          initPost()
-         init()
+         prepareCurrentFeed()
+         makeRequest()
+         postReply()
+         openPostInWebview()
     }
 
-    private fun init() {
+    private fun makeRequest() {
```

Comment:
Function name is too abstract. Rename to smth like `fetchCurrentFeed()` or `makeGetFeedRequest()` or smth more specific